widget.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Simple Disk Stats",
+  "name": "Drive Space",
   "description": "Minimal occupied space status.",
-  "author": "Andrew Pepin",
-  "email": "paradiddley@gmail.com"
+  "author": "David Idol",
+  "email": "david.idol@gmail.com"
```

Comment:
Wasn't sure what to put here - can there be two authors? Also happy to keep you on here (really doesn't matter to me :))